### ISSUE TITLE
Create special config version for all this, so as to allow releasing plain MSW versions.

### ISSUE THREAD

**Issue by mmehr2** [mmehr2/Msw4#7]
The MSW project build is set up to have multiple VS configurations for each version. There is DebugU, which I am concentrating on, with ReleaseU for release versions. These should be plain versions though.

We want a DebugRemote and ReleaseRemote pair of configs, and put the U versions back to their state before adding all this (but with my improvements). Might take a while to sort it all out tho.

No need to use Git branches, I believe.

**Comment by mmehr2** [comment]
Won't be needed until final delivery.

**Comment by mmehr2** [comment]
Becomes more urgent as I begin to modify the UI code. SO far all changes are localized to the Comm.cpp calling tree, and the build files. Verified this on Git.

**Comment by mmehr2** [comment]
My current thinking is to finish this before tackling UI modifications. Although intended to be only for the RemoteDialog class, changing the UI does modify global resource lists, header constants, etc. It will just be cleaner to have the RelaseR and DebugR configurations done. I also need to totally remove the XMPP code as well (libjingle) and its dependencies from the main production configuration (and remote for that matter, too). Main release versions do not use _REMOTE flag, so will build without it anyway. But the project file still refers to the source. 

Perhaps there is no problem to keep it in for reference, I'll put this off for now as Low Prio.

**Comment by mmehr2** [comment]
This will be included in the work for issue #19 and closed here. It's not a useful redundancy to maintain.

**Comment by mmehr2** [comment]
Just to update that I have completed this work today. There are now ReleaseU and DebugU versions that are back to their simple build configurations, with no _REMOTE flag. All configuration changes will be made in the ReleaseR and DebugR ones just created. Also, I've marked the branch point in the code repo if we need to go back and make that version for release without the remote sources. No need to branch until things get settled with the production release process.
